messages/core_write_access.md

```diff
@@ -0,0 +1,14 @@
+Hello {NAME},
+
+{some information about why they are getting access now}
+
+You can see the formal description of the subpackage maintainers role at http://www.astropy.org/team.html.  But there are some other guidelines to keep in mind, which I'm describing here.  In general you should only merge PRs that are in the domain of your subpackage(s).  This is a guideline rather than a firm rule: if you see something that's straightforward and obvious (e.g., a minor typo fix that's already been approved and the tests have passed but no one hit the merge button), you should feel free to help with that sort of thing even in other packages. Similarly, in areas where there's conflict/uncertainty, it's best to defer to a more senior maintainer (if there is one) or if absolutely necessary the Coordination Committee... but in cases where there's no obvious reason to question it (particularly if the lead is the PR author), it is also fine to merge things when it seems reasonable.
+
+Relatedly, if you haven't already, you should familiarize yourself with the maintainer guidelines: https://docs.astropy.org/en/latest/development/workflow/maintainer_workflow.html.  Please respond indicating you've seen these so we're sure you're aware of them.
+
+You should also feel free to help with labeling new PRs or issues as they come in - that's a task all of the subpackage maintainers share, since it's only people with write privileges can do.  So don't feel at all shy about labeling these, even outside your subpackage(s).
```

Comment:
maybe also add a feel free to "review PRs". We need more maintainers to review PRs without the fear to step on toes.